assets/xml/scenes/dungeons/bdan.xml

```diff
@@ -1,7 +1,7 @@
 <Root>
     <File Name="bdan_scene" Segment="2">
         <Cutscene Name="gJabuIntroCs" Offset="0x155E0"/>
-        <Cutscene Name="gJabuSapphireRetrieveUnusedCs" Offset="0x13080"/>
+        <Cutscene Name="gRutoObtainingSapphireUnusedCs" Offset="0x13080"/>
```

Comment:
this should probably keep the scene prefix cause its in the scene. the whole "Status quo" thing from the other day.